h2o-k8s/tests/clustering/h2o-python-clustering-template.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: h2o-assisted-python
+  labels:
+    app: h2o-assisted-python
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: h2o-assisted-python
+  template:
+    metadata:
+      labels:
+        app: h2o-assisted-python
+    spec:
+      containers:
+        - name: "h2o-assisted-python"
+          image: "harbor.h2o.ai/opsh2oai/h2o-3-k8s-runner:4"
+          command: ['/bin/bash', '-c', 'python assisted-clustering.py --namespace default h2o-assisted']
+          volumeMounts:
+            - name: python-clustering-script
+              mountPath: "assisted-clustering.py"
+          resources:
+            limits:
+              cpu: '100m'
+              memory: "256Mi"
+            requests:
+              cpu: '100m'
+              memory: "256Mi"
+          env:
+          - name: H2O_ASSISTED_CLUSTERING_REST
+            value: 'True'
+      volumes:
+        - name: python-clustering-script
+          hostPath:
+            path: $H2O_BASE/h2o-k8s/tests/clustering/assisted-clustering.py
+            type: File
+---
+apiVersion: v1
+kind: Service
+metadata:
+ name: h2o-assisted
+ namespace: default
+ labels:
+   app: h2o-assisted
+spec:
+ type: ClusterIP
+ clusterIP: None
+ selector:
+   app: h2o-assisted
+ ports:
+   - protocol: TCP
+     port: 80
+     targetPort: 54321
+---
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
Why not `networking.k8s.io/v1`? OpenShift?

Reply:
Yes.